evals/benchmark/README.md

```diff
@@ -12,12 +12,171 @@ This Tool provides a microservice benchmarking framework that uses YAML configur
 
 ## Table of Contents
 
+- [Workload Deployment in Kubernetes](#workload-deployment-in-kubernetes)
+  - [Prerequisites](#prerequisites)
+  - [Deployment](#deployment)
+    - [1. Deploy with CPUs only](#1-deploy-with-cpus-only)
+    - [2. Deploy with Gaudi Devices](#2-deploy-with-gaudi-devices)
+    - [3. Deploy with Multiple Instances](#3-deploy-with-multiple-instances)
+    - [4. Deploy with a Specific Namespace](#4-deploy-with-a-specific-namespace)
+    - [5. Deploy with Proxy Support](#3-deploy-with-proxy-support)
+  - [Destroy](#destroy)
 - [Installation](#installation)
 - [Usage](#usage)
 - [Configuration](#configuration)
   - [Test Suite Configuration](#test-suite-configuration)
   - [Test Cases](#test-cases)
 
+## Workload Deployment in Kubernetes
```

Comment:
A little confusing, you use plural, as in CPU**s**, Gaudi instance**s**, why call out separately "multiple instances"?  Let the Kubespray and kubeadm instructions help the user with multi-node cluster setup. Let us also stick to nodes for the compute resources, and mention they could be bare metal machines with OS or virtual machines. Let us reserve instances for container and/or pod instances.

Your multi-node cluster section is to educate the user about model download overhead to multiple nodes can be reduced and secondly how to control where some workloads can land with labels.

Reply:
s/Deploy with Multiple Instances/Configure Multiple Pod Instances.

"Your multi-node cluster section is to educate the user about model download overhead to multiple nodes can be reduced and secondly how to control where some workloads can land with labels." For `replicaCount`,  do you suggest removing it? While enabling Horizontal Pod Autoscaler (HPA) for scaling is important, it falls under tuning rather than being out-of-the-box (OOB) functionality. I recommend retaining the replicaCount field as a reference for others, especially when we remove the 1/2/4/8 manifests."